src/scripts.js

```diff
@@ -1,19 +1,76 @@
 
-import {fetchAllData} from './APICalls';
+import {fetchAllData, fetchSingleUser, postData} from './APICalls';
 import Hotel from './classes/hotel';
+import domUpdates from './domUpdates'
 import './css/base.scss';
+import Customer from './classes/customer';
 
+// Global variables
 let hotel;
+let currentUser;
+let sum;
+let rooms;
 
-document.querySelector('button').addEventListener('click', fetchTheData())
+//QuerySelectors
+const searchForm = document.querySelector('form');
 
-function fetchTheData() {
-  let roomsPromise = fetchAllData('rooms');
-  let customersPromise = fetchAllData('customers');
-  let bookingsPromise = fetchAllData('bookings');
-  
-  Promise.all([roomsPromise, customersPromise, bookingsPromise])
-    .then(data => hotel = new Hotel(data[0], data[1], data[2]))
-    .then(() => console.log(hotel))
+
+// Functions
+const loadPage = () => {
+  fetchTheData()
+  .then(data => {
+    hotel = new Hotel(data[0], data[1], data[2])
+    addInfo()
+    defineUser(50)
+    singleCustomerInfo()
+  })
+}
+
+const addInfo = () => {
+  hotel.addRooms();
+  hotel.addBookings();
+  hotel.addCustomers();
+}
+
+const singleCustomerInfo = () => {
+  defineUser(50);
```

Comment:
In your fetchTheData function above, you've called defineUser(50) already and then you call this singleCustomerInfo() function that contains defineUser(50) as well. You can delete one of them.